### ISSUE TITLE
Frame Scale Solve Calculation

### ISSUE THREAD

**Issue by astroman68** [indigo-astronomy/indigo_imager#7]
Rumen,

Not sure if this is a bug or if there is something setup incorrectly. If there is, I haven't been able to find it. 

When I do a solve using my Atik 428ex, I get the correct frame size and rotation, but the scale is reading wrong showing ~144"/px. It should be 2.4"/px. To check if maybe there was a calculation bug in the code, I did a solve using my guide camera, Altair 130M and got the correct scale. 

I have checked everything I can find in indigo referencing the camera specs as well as lens profile aperture & focal length. It seems to be setup correctly.

One interesting thing is if you convert 144"/px to arc-minutes, you do get 2.4'. So its almost like I got something set wrong. I noticed in indigo everything is in CM instead of MM, which is what most measurements are in that I have seen. So I made sure my aperture and focal length was in CM. At one time I had it set in MM not noticing it should be in CM, but even after setting them to CM, the scale didn't change.

So, not a big deal as it doesn't really effect anything, but if it is a bug I thought you would want to know. If its something I have set wrong, can you point me in the right direction? I have attached a pic of a solve.

![atik-frame-statistics](https://user-images.githubusercontent.com/83990821/147182814-acffcc7b-b9e9-4439-9e47-da9d0ece2a69.png)

**Comment by rumengb** [comment]
Can you provide an image that is solved like this?
I can not guess what might be wrong from the screenshot only...
Do you have any fists file you can send me?

**Comment by rumengb** [comment]
Oh by the way, just noticed it is 2.4"/px but the equivalent of 2.4'/px is displayed. We had such bug in indigo some time ago regarding the scale of the solution. But I think it should be resolved in the recent releases. Which version of indigo server you are using? Are you running the latest indigo? Any way a fits file will help in any case if the bug is till there it will help us to fix it...

**Comment by astroman68** [comment]
Thanks Rumen. I'm using the latest from the repo on Ubuntu server. INDIGO server v.2.0-166 built on Nov 30 2021 10:11:19. Link to fits file... [https://gitlab.com/astro-man/general-share/-/blob/d22efbfb1ddd845ef9b6d49e008a0bf0dfcd1531/IC434_2021-12-09_Light_138.fits](url)

**Comment by rumengb** [comment]
Yes, there is an issue in indigo itself it is not Ain. Issue is fixed in the repo. it will be available in the next indigo release!
![image](https://user-images.githubusercontent.com/20597893/147273969-3a9fb1bc-abf5-4b08-bbb0-5f76777a8625.png)
